portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/cpusersandorganizations/Organizations.testcase

```diff
@@ -706,6 +706,67 @@ definition {
 		}
 	}
 
+	@description = "Use case for LPS-198960.Organization users and sub organizations are not mix across pages using interator button"
+	@priority = 4
+	test CanListedUsersanddSubOrganizationsAcrossPages {
+		task ("Given an Organization with two Users") {
+			JSONOrganization.addOrganization(organizationName = "Organization for Users");
+
+			for (var user : list "A,B") {
```

Comment:
You can also call them user1, user2 instead of A,B

Reply:
same of  above for Sub Organizatons